Edits to PGD 5.5 release (#5610)
EnterpriseDB/docs: product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

-   `config_value` &mdash; New value to be set for the given key.

`config_value` will be parsed into the data type appropriate for the option.

> Note that some parameters can be applied only to the top-level node group. These are denoted in the table with Top appearing in the Groups column. Where the parameter can be applied to the top-level node group and subgroups, All appears in the Groups column. Sub in the Groups column means the parameter can be applied only to subgroups.

Author: ```suggestion
Some parameters can be applied only to the top-level node group. For these parameters, the Groups column contains Top. Some parameters can be applied to the top-level node group and subgroups. For these parameters, the Groups column contains All. When a parameter can be applied only to subgroups, the Groups column contains Sub.
```
